common/djangoapps/dark_lang/api.py

```diff
@@ -0,0 +1,199 @@
+"""
+Support code for dark-launching languages.  These languages won't be used
+when determining which translation to give a user based on their browser
```

Comment:
@cahrens Originally I was planing on deleting the middleware file, but later found that the function of middleware was still needed.  

I can change the name of the file, but believe the functionality included does not belong in the middleware file.  Once I remove the clean_accept_headers method and supporting methods back to middleware, there is a clear separation.  The remaining methods are used by the views.py file to process the incoming GET and POST requests.